src/Common/src/System/Net/SafeCloseSocket.Windows.cs

```diff
@@ -64,13 +64,8 @@ public ThreadPoolBoundHandle GetOrAllocateThreadPoolBoundHandle(bool trySkipComp
                             CloseAsIs();
                             throw;
                         }
-
-                        // Try to disable completions for synchronous success, if requested
-                        if (trySkipCompletionPortOnSuccess &&
-                            CompletionPortHelper.SkipCompletionPortOnSuccess(boundHandle.Handle))
-                        {
-                            _skipCompletionPortOnSuccess = true;
-                        }
+                        
+                        _skipCompletionPortOnSuccess = false;
```

Comment:
We shouldn't need any changes to production code.

@schen381 let's take this offline as it requires more investigation regarding missing APIs.